src/accordion/accordion.ts

```diff
@@ -195,7 +208,7 @@ export interface NgbPanelChangeEvent {
           <ng-template [ngTemplateOutlet]="panel.headerTpl?.templateRef || t"
                        [ngTemplateOutletContext]="{$implicit: panel, opened: panel.isOpen}"></ng-template>
         </div>
-        <div id="{{panel.id}}" role="tabpanel" [attr.aria-labelledby]="panel.id + '-header'"
+        <div id="{{panel.id}}" (ngbRef)="panel.panelDiv = $event" role="tabpanel" [attr.aria-labelledby]="panel.id + '-header'"
```

Comment:
Oh, Im so not convinced by passing id as the event on init :)

Reply:
Note that this is not an id, it is the reference to the element itself, thus removing the need of an id.

Reply:
Sure. But on the other hand I just thought that it also avoids doing `querySelector` for each panel on every `ngAfterContentChecked`...

Reply:
`ngbRef` seems to be inspired from [callback refs in react](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs).
We could instead use [refs object as in react](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) which would perhaps convince you more than the callback form.

Reply:
I see, and on my side I can't really find a simpler way to pass the element ref once during the initialization

Reply:
maybe I would prefer passing a function instead of using event emitter